app/models/ext_management_system.rb

```diff
@@ -686,7 +686,7 @@ def wait_for_ems_workers_removal
   end
 
   def orchestrate_destroy_queue(task_id, queue_options = {})
-    self.class._queue_task('orchestrate_destroy', [id], task_id, queue_options)
+    self.class._queue_task('orchestrate_destroy', [id], task_id, queue_options.reverse_merge(:msg_timeout => 3_600))
```

Comment:
See line 695 for "requeue when not yet paused" case not setting msg_timeout previously...